src/templates/specific-profile.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react'
+import { Link, graphql } from 'gatsby'
+import Layout from '../components/Layout.js'
+import PreviewCompatibleImage from '../components/PreviewCompatibleImage.js'
+import Tabs from '../components/Tabs/Tabs.js'
+import { FaCheck, FaUser, FaList, FaExclamationTriangle } from 'react-icons/fa'
+import LinesEllipsis from 'react-lines-ellipsis'
+
+const ProfileTemplate = ({ data }) => (
+    <Layout>
+      <section className="section">
+        <div className="columns is-centered is-multiline">
+            <div className="column is-full-mobile is-two-thirds-tablet is-half-desktop" style={{maxWidth:'400px'}}>
+                <PreviewCompatibleImage imageInfo={data.strapiProfile.profile_image}/>
+            </div>
+          <div className="column is-full-mobile is-two-thirds-tablet is-half-desktop">
+              <div style={{display:'inline', padding: '5%'}}>
+                <h1 className="has-text-weight-bold is-size-2">{data.strapiProfile.profile_name}</h1>
+                <div className="is-italic">
+                {data.strapiProfile.profile_location}
+                </div>
+                <hr className="horizontal-rule" />
+                <div className="column is-12">
+                    {data.strapiProfile.profile_bio}
+                </div>
+              </div>
+            </div>
+        </div> <br/>
+
+        <div>
+        <Tabs>
+          <div label="Skills" icon="FaCheck">
+            <center>{data.strapiProfile.skills.map(skill => (
+                <div>
+                <br/>
+                <div>
+                  {skill.skill_name}
+                  <br />
+                </div>
+                </div>
+            ))}</center>
+            <br /><br />
+          </div>
+
+
+          <div label="Experience" icon="FaUser">
+
+              <div className="columns is-multiline is-centered">
+                {data.strapiProfile.experiences.map(experience => (
+                    <div className="column is-12">
+                        <div className="has-text-weight-bold is-size-5">
+                            {experience.experience_title} <br />
+                        </div>
+                        <div>
+                            {experience.experience_description}
+                        </div>
+                    </div>
+                ))}
+              </div> <br /><br />
+          </div>
+
+
+          <div label="Endorsements" icon="FaList">
```

Comment:
What's the plan with Endorsements? Should we hide them or actually add them in strapi? Or are we fine with this approach?